internal/controllers/clusterctl/config.yaml

```diff
@@ -48,8 +48,8 @@ data:
       type:         "ControlPlaneProvider"
 
     # Addon providers
-    - name:         "fleet"
-      url:          "https://github.com/rancher-sandbox/cluster-api-addon-provider-fleet/releases/v0.7.0/addon-components.yaml"
+    - name:         "rancher-fleet"
```

Comment:
might need rebase after https://github.com/rancher/turtles/pull/1190

Reply:
Since this change is a part of this PR, I think we just merge it first. Updatecli will get closed automatically.

Reply:
Ah, there are some other provider bumps. Ok, let’s see once it gets there

Reply:
Should be rebased, thanks.